# This message contains additional information from the Ainstein US-D1 radar.
sensor_msgs/Range range
uint8 snr # Signal to noise ratio
float32 variance # Variance according to datasheet [m^2]
